core/services/job/job_orm_test.go

```diff
@@ -191,6 +193,18 @@ func TestORM(t *testing.T) {
 		_, err = orm.CreateJob(context.Background(), &jb, jb.Pipeline)
 		require.NoError(t, err)
 	})
+
+	t.Run("creates webhook specs along with external_initiator_webhook_specs", func(t *testing.T) {
+		tree, err := toml.LoadFile("../../testdata/tomlspecs/webhook-job-spec-external-initiators.toml")
```

Comment:
Consider adding a webhook spec to testdata/testspecs/v2_specs.go (see GenerateVRFSpec as an example of an in-memory spec generate function with overridable defaults) to get away from file reading in UT. Ideally the tomlspecs ones are just used for a top level integration test

Reply:
Might not even need MustInsertWebhookSpec if we have that you could just 
```
ws, err := webhook.ValidatedWebhookSpec(testspecs.GenerateWebhookSpec(WebhookSpecParams{}).Toml(), mgr)
require.NoError(t, err)
require.NoError(t, db.Create(ws).Error)
```

Reply:
Nice, didn't know about this spec generation file. This is much better and allows me to drop the fixtures :+1: